bot/cogs/base_converter_cog.py

```diff
@@ -0,0 +1,64 @@
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+class BaseConverterCog(commands.Cog):
+
+    def __init__(self, bot) -> None:
+        self.bot = bot
+
+    @commands.group(aliases= ['convert','baseconvert'])
+    async def bconvert(self, ctx, *args):
+        """
+        A simple base converter that takes in a base number an a value, then displays the value in binary, octal, decimal, and hexadecimal
+        
+            Examples:
+            bconvert [bin / binary] [11 / 0b11]
+            bconvert [dec / decimal] [99]
+            bconvert [hex / hexadecimal] [FF / 0xFF]
+            bconvert [oct / octal] [77 / 0o77]
+        """
+        base = args[0]
+        number = args[1]
+        if base is None or number is None:
+            await ctx.send('Missing arguments! Use the help command for examples')
+            return
+
+        if base == 'bin' or base == 'binary':
```

Comment:
having them as grouped sub commands was def the right approach, its much more extensible. then the base command can list out what it needs